cosmic-client/src/main/webapp/scripts/network.js

```diff
@@ -850,7 +852,7 @@
                                             //include currently selected vpc offering to dropdown
                                             items.push({
                                                 id: args.context.vpc[0].vpcofferingid,
-                                                description: args.context.vpc[0].vpcofferingdisplaytext + " (Current offering)"
+                                                description: args.context.vpc[0].vpcoffering + " (Current offering)"
```

Comment:
The original line is the correct way, `vpcoffering` doesn't exists and shows `undefined` in the pulldown list